src/web/routes/application/steps/address/select-address/select-address.js

```diff
@@ -3,6 +3,8 @@ const { transformAddress } = require('./adapters')
 const { addressContentSummary } = require('../content-summary')
 const { requestBody } = require('../request-body')
 const { stateMachine, actions } = require('../../../flow-control')
+const { validate } = require('./validate')
+const { setAdditionalDataForStep } = require('../../../flow-control/session-accessors')
```

Comment:
This should be exported from `flow-control` - see line 5

Reply:
done, thanks